run-tests.sh

```diff
@@ -20,8 +20,12 @@ check_sphinx () {
     sphinx-build -qnNW docs docs/_build/html
 }
 
+check_lint () {
+    cd reana-ui && yarn lint && cd ..
```

Comment:
Minor: another option would be to use parentheses in commands. In this way we don't have to go back (`cd ..`). Command would look like this: `(cd reana-ui && yarn lint)`. Same could be applied for `check_prettier` and `check_js_tests`. WDYT?

Reply:
Make sense, didn't know the subshell functionality 👍